### PR TITLE
Fix issue 683

### PR DESCRIPTION
**Pull request by marzn**
This fixes issue #683 

----

Make sure these boxes are checked before submitting your PR -- thank you!

- [x] Added an entry into `CHANGELOG.md` if you have changed SpotBugs code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 ### Fixed
 
 * Potential NPE in test-harness-core ([#671](https://github.com/spotbugs/spotbugs/issues/671))
+* Support project path with spaces ([#683](https://github.com/spotbugs/spotbugs/issues/683))
```
Comment (KengoTODA) [inline CHANGELOG.md]: Good. Could you add `in test-harness-core`, then reader can easily understand this fix is for that module.

**Comment by KengoTODA** [comment]
If possible, please try to create an unit test case to reproduce problem.

**Comment by marzn** [comment]
@KengoTODA because this is related to the absolute path of a project, i think this would be more a integration test. Of course we can check if `AnalysisRunner.createTempJar()` is producing the correct output, but this bug can occur in various methods, if the path is not decoded or encoded twice.

**Comment by marzn** [comment]
Anyway, I had a look at the `AnalysisRunner`. Only for the `createTempJar()` unittest it would be a lot of code to setup the test. On the other hand this does not fulfill the requirements in my eyes. So I stopped working on the unittest.

**Pull request merged**